core/mastodon/src/main/java/br/com/orcinus/orca/core/mastodon/auth/authentication/activity/MastodonAuthenticationActivity.kt

```diff
@@ -80,6 +81,7 @@ class MastodonAuthenticationActivity : ComposableActivity() {
   private fun authenticate() {
     viewModel.request {
       instance.authenticator.receive(it)
+      MastodonNotificationService.bind(applicationContext)
       finish()
     }
   }
```

Comment:
A connection should be bound to the service every time the app is open, the user is authenticated and one has not been bound yet.